readyset-example/app.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import argparse
+import psycopg2
+import time
+import numpy as np
+
+parser = argparse.ArgumentParser(
+    description="test performance of ReadySet vs. a backing Postgres database")
+parser.add_argument("--query",
+                    required=False,
+                    help="query to execute")
+parser.add_argument("--repeat",
+                    type=int,
+                    help="number of times to run the query",
+                    default = 20)
+parser.add_argument("--url",
+                    required=True,
+                    help="connection URL for ReadySet or Postgres")
+args = parser.parse_args()
+
+query = "SELECT count(*) FROM title_ratings JOIN title_basics ON title_ratings.tconst = title_basics.tconst WHERE title_basics.startyear = 2000 AND title_ratings.averagerating > 5"
+
+print(args.url)
```

Comment:
@Peeeekay any risk of leaking sensitive credentials? I think their password could leak out here

Reply:
oh good catch, i was doing some debugging  - forgot to remove this. I would not recommend folks to put password in plain-text in the files - they should probably leverage env variables; this is just a prototype for them to see how kurtosis can be helpful. I can bring this up in readme as well.